testplan/testing/multitest/result.py

```diff
@@ -1871,6 +1871,42 @@ def matplot(self, pyplot, width=2, height=2, description=None):
             description=description
         )
 
+    @bind_entry
+    def graph(self, graph_type, graph_data, description,
+              series_options, graph_options):
+        """
+        Displays a Graph in the report.
+
+         .. code-block:: python
+
+             result.graph('Line',
+                          {
+                              'graph 1':[{'x': 0, 'y': 8},{'x': 1, 'y': 5}]
+                          },
+                          description='Line Graph',
+                          series_options={'graph 1':{"colour": "red"}},
+                          graph_options=None
+              )
+
+        :param graph_type: Type of graph user wants to create
+        :type graph_type: ``str``
+        :param graph_data: Data to plot on the graph, for each series.
+        :type graph_data: ``dict[str, list]``
+        :param description: Text description for the graph.
+        :type description: ``str``
+        :param series_options: customisation parameters for each series
```

Comment:
please document series & graph options

Reply:
This is already documented in detail under the Assertions section. I agree that adding a little more detail to the API docs would also be useful (it's fine to duplicate information slightly in the API docs).